agent/lib/kontena/service_pods/creator.rb

```diff
@@ -159,6 +159,7 @@ def ensure_image(name)
       # @param [Docker::Container] service_container
       # @return [Boolean]
       def service_uptodate?(service_container)
+        return false unless service_container.running?
```

Comment:
Yes, but container is swapped also on any scheduler change (for example if image has been updated). Anyway, I'll change this to "start" so that we can move on.